### ISSUE TITLE
[BUG] chat image of item/spell is lost after clicking (dis)advantage buttons

### ISSUE THREAD

**Issue by maralasar** [MangoFVTT/fvtt-ready-set-roll-5e#59]
**Description**
After I rolled an attack with either a spell or an item and clicked the (+) or (-) in chat to roll for advantage or disadvantage the image of the item or spell is lost and replaced with the foundry default image. If Advantage and Disadvantage is triggered via Shift or CRTL the image is displayed as expected

**Steps to Reproduce**
1. Roll an attack with an Item or Spell
2. Click the advantage (+) or disadvante (-) button in the chat message

**Versions**
 - Module: [1.3.4]
 - dnd5e: [2.0.2]
 - Foundry: [10.285]

**Other Modules**
None. Deactivated everything but "Ready Set Roll" and "libWrapper".

EDIT:
The problem is persistent through different Browsers (Chrome, Chromium, Firefox) and does not care for where the image is save or the file format (only tried .png and .jpg). The Console does not show any errors!

**Comment by MangoFVTT** [comment]
I haven't been able to reproduce this at all on my end, with or without modules. 🤔 Does this happen for you with all items? If not, can you provide specific details about the items that are having the problem. Also, can you check if this also happens with Ability checks/saves/skills as well?

![image](https://user-images.githubusercontent.com/110994627/190484903-005e4b75-0453-4ce0-b5b9-b697379671e4.gif)

**Comment by maralasar** [comment]
I have attached files for the item and the occuring problem!

![firefox_xLLt9ODR5a](https://user-images.githubusercontent.com/113531659/190486634-b14e5c8a-3fe5-490b-81ba-9554898de7c3.png)

![firefox_f4sEq8Db7a](https://user-images.githubusercontent.com/113531659/190486569-d9f3c096-ab42-452d-94e0-ddb1aa077b85.gif)

**Comment by maralasar** [comment]
I tried it with a new world, new actor and new item and everything is working a expected! 
It seems the problem was on my side with actors updated from v9 to v10.
Weirdly, after I changed the name of the actor in the old world, making problems the new icon for copying the actors id appeared right beside its name in the topbar of the character sheet and now everything is working, even in the old word!

![firefox_u4QpEQrS0V](https://user-images.githubusercontent.com/113531659/190490840-41d50d82-9ec2-4d3e-8ad5-bd7520f736fc.png)
